src/Microsoft.OData.Edm/Vocabularies/CommunityVocabularyConstants.cs

```diff
@@ -0,0 +1,17 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CommunityVocabularyConstants.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Edm.Vocabularies.Community.V1
+{
+    /// <summary>
+    /// Constant values for Community Vocabularies
+    /// </summary>
+    public static class CommunityVocabularyConstants
+    {
+        /// <summary>OData.Community.V1.UrlEscapeFunction </summary>
+        internal const string UrlEscapeFunction = "Org.OData.Community.V1.UrlEscapeFunction";

```

Comment:
>internal [](start = 8, length = 8)

Either make this public or make the class internal.  Unless we think there is a client scenario that requires it, I would make the class internal -- we can always make it public later.

Reply:
updated.

---
In reply to: [231620168](https://github.com/OData/odata.net/pull/1307#discussion_r231620168) [](ancestors = 231620168)